Formula/ansible@2.9.rb

```diff
@@ -0,0 +1,693 @@
+class AnsibleAT29 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Automate deployment, configuration, and upgrading"
+  homepage "https://www.ansible.com/"
+  url "https://files.pythonhosted.org/packages/f0/03/4547c6afee926b842a288b12036a7bddce4f7f661b6797414eaee6ac2ccc/ansible-2.9.17.tar.gz"
+  sha256 "d78b46d647d91dd478ba7f43baf36ffa27f9a5992b826b16a6cc15cf993f19dc"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url "https://github.com/ansible/ansible"
+    regex(/^v(2.9.\d+)$/i)
```

Comment:
```suggestion
    regex(/^v(2\.9\.\d+)$/i)
```
Sorry, just noticed this. We need to escape `.` or else that will match any character.

Reply:
Thanks for catching that!